compiler/testData/diagnostics/tests/controlStructures/whenWithNothingTypedSubject.fir.kt

```diff
@@ -0,0 +1,8 @@
+// !DIAGNOSTICS: -UNREACHABLE_CODE
+
+typealias MyNothing = Nothing
+
+fun foo(n: Nothing, n2: MyNothing) {
+    val a: Unit = when(n) {}
+//    val b: Unit = when(n2) {}
```

Comment:
The FIR and FE 1.0 files would be identical except that here you have this line commented out. I don't know if you meant to just delete the line completely. Don't think you need it.

Reply:
Oops! Let me remove this file.